src/routes.js

```diff
@@ -130,7 +130,9 @@ module.exports = [
     handler: (request, reply) => {
       reply({status: 'ok'});
       if (process.env.NODE_ENV === 'testing'){ return; }
-      sendMail(request.payload, function(err, info){
+      var payload = JSON.parse(request.payload);
+      var emailBody = JSON.stringify(payload[1]);
```

Comment:
See my comment on the xhr request.
You want:
```js
const { emalAddress, emailBody } = payload
```
at the top of this function. (you dont have to use es6 though )